fast_llm/data/dataset/gpt/sampled.py

```diff
@@ -92,6 +95,11 @@ def __init__(
         self._truncate_documents = sampling.parameters.truncate_documents
         self._device = torch.device("cuda" if self._config.gpu else "cpu")
 
+        if self._indexed_dataset.has_images and self._truncate_documents:
+            raise RuntimeError(
+                "Truncating documents with images is not yet supported. Please turn off truncation to use images."
+            )
```

Comment:
what does that mean in practice? documents with images that are longer than the sequence length are discarded?